client/src/helpers/providers/extractor.ts

```diff
@@ -52,49 +52,54 @@ export class DataExtractor {
   }
 
   public getPractitionerMetaData(id: string): PractitionerMetaData | undefined {
-    if (this.data.practitionersData.entry == null) {
-      return undefined;
-    }
-    for (let i = 0; i < this.data.practitionersData.entry.length; i += 1) {
-      if (this.data.practitionersData.entry[i].resource.entry == null) {
-        continue;
-      }
+    if (this.data.practitionersData.entry) {
+      for (let i = 0; i < this.data.practitionersData.entry.length; i += 1) {
+
+        const resource = get(this.data, `practitionersData.entry[${i}].resource`, {})
```

Comment:
Using lodash get to ignore the `this.data.practitionersData.entry[i].resource.entry == null` check condition